agent/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
```

This will change the default Docker compose project name from "agent" (folder name).

```suggestion
name: bleemeo-community-edition-agent

services:
```

We don't use just "bleemeo-community-edition" because base and/or high-availability would use the same name and docker compose would complain about "orphan containers"